aries-backchannels/acapy/Dockerfile.acapy-main

```diff
@@ -22,11 +22,13 @@ COPY python python
 COPY acapy acapy
 COPY data ./
 
-# aca-py is no longer in /usr/local/bin. 
-# The Backchannel is looking for it in ./bin, copy file and create a link to it in ./bin
+# Poetry installation install and build to create aca-py bin file
+RUN pip install --no-cache-dir poetry
```

Comment:
We don't need these poetry lines in the dockerfiles.

```
# Poetry installation install and build to create aca-py bin file
RUN pip install --no-cache-dir poetry
RUN poetry build
```

can be removed from both dockerfiles. `pip` will install `/usr/local/bin/aca-py` so we just need the step that creates the symbolic link.  when this issue first popped up, there was some kind of glitch with pip and the new acapy distribution that was not installing the `bin/aca-py`... it magically started working a week later. so all that workaround to use poetry in the AATH images ended up being unnecessary.

Reply:
Also, we are in a weird spot with the redis events plugin, the plugin requires python 3.7 but that is too old for a lot of libs... so we need [aries-acapy-plugin-redis-events](https://github.com/bcgov/aries-acapy-plugin-redis-events.git) to be updated... then we can update the Dockerfile.acapy-redis image to be `python:3.9-slim-bullseye` based.

Reply:
It didn’t start working again — it is still not working in AATH.  I reversed your change and it still fails in the same way it was before.  I think it makes sense, since in changing to poetry we removed the `bin/aca-py` file. AFAIK, it is created by poetry, hence the additions.  However, I see from @nodlesh found, that does not work in the build, which is strange because AFAICS, it is working (I think…) when I use the `./manage runset acapy -r` (`-r` means rebuild…) does seem to work.  Still investigating...

I’d like to have us generate the aca-py bin from ACA-Py vs. using a copy, but its not that big a deal.